Dockerfile

```diff
@@ -116,10 +118,10 @@ WORKDIR /icgc
 # Set path defaults as environmental variables
 #
 
-ENV ICGCGET_ICGC_PATH = /icgc/icgc-storage-client/bin/icgc-storage-client
-ENV ICGCGET_GDC_PATH = /icgc/gdc-data-transfer-tool/gdc-client
-ENV ICGCGET_EGA_PATH = /icgc/ega-download-demo/EgaDemoClient.jar
-ENV ICGCGET_CGHUB_PATH = /icgc/genetorrent/bin/gtdownload
-ENV ICGCGET_PDC_PATH = /local/bin/aws
+ENV ICGCGET_ICGC_PATH=/icgc/icgc-storage-client/bin/icgc-storage-client
+ENV ICGCGET_GDC_PATH=/icgc/gdc-data-transfer-tool/gdc-client
+ENV ICGCGET_EGA_PATH=/icgc/ega-download-demo/EgaDemoClient.jar
+ENV ICGCGET_CGHUB_PATH=/icgc/genetorrent/bin/gtdownload
+ENV ICGCGET_PDC_PATH=/usr/local/bin/aws
 
 ENTRYPOINT ["icgc-get"]
```

Comment:
This is what I was thinking about customizing by wrapping for a Docker bootstrap script.